src-self-hosted/translate_c.zig

```diff
@@ -3018,6 +3018,8 @@ fn transCompoundAssignOperator(rp: RestorePoint, scope: *Scope, stmt: *const Zig
             return transCreateCompoundAssign(rp, scope, stmt, .AssignSubWrap, .MinusPercentEqual, "-%=", .SubWrap, .MinusPercent, "-%", used)
         else
             return transCreateCompoundAssign(rp, scope, stmt, .AssignSub, .MinusPercentEqual, "-=", .Sub, .Minus, "-", used),
+        .DivAssign => return transCreateCompoundAssign(rp, scope, stmt, .AssignDiv, .SlashEqual, "/=", .Div, .Slash, "/", used),
+        .RemAssign => return transCreateCompoundAssign(rp, scope, stmt, .AssignMod, .PercentEqual, "%=", .Mod, .Percent, "%", used),
```

Comment:
Signed integer divisions have to use `@divTrunc` and `@rem` builtin functions. Take a look at `transBinaryOperator` for help on doing the checks.
Also it seems that `transBinaryOperator` is checking `!cIsUnsignedInteger(qt)` when it should be checking `cIsSignedInteger(qt)` so you could also fix that.

Reply:
Ah, I'll switch it to use those!